Packages/com.chisel.editor/Chisel/Editor/Editor/Scene/Handles/PointDrawing.cs

```diff
@@ -98,6 +104,8 @@ public static void Reset()
 
             if (s_StartIntersection != null)
             {
+                if (!dragArea.Contains(mousePosition))
+                    return null;
```

Comment:
Tried to make drawing a shape that is larger than your sceneview work, but something weird is happening on the side of unity. Need to investigate this before I can turn it back on. For now just ignore any mouse movement outside the sceneview